lib/mongoid/config.rb

```diff
@@ -25,6 +25,7 @@ module Config
     option :duplicate_fields_exception, default: false
     option :use_activesupport_time_zone, default: true
     option :use_utc, default: false
+    option :atomically_join_context_default, default: false
```

Comment:
Can this be renamed to just `join_contexts` please so that the same setting will be usable for transactions.